crates/core/src/factories/core_space.rs

```diff
@@ -355,6 +355,8 @@ impl Space for CoreSpace {
 
                         // add it to bootstrapping.
                         bootstrap.put(info);
+                    } else {
+                        tracing::trace!("Not updating agent info because we don't have a current url");
```

Comment:
Isn't that worth a warning or even error?

Reply:
Possibly, we won't normally see it on startup I don't think. We generally seem to avoid giving out warnings for things that are likely to correct themselves and that the user can't fix. I'd quite like to make this a warning or at least info. Given that you phrased this request as a question, what do you think?

Reply:
Yes, info or warning is good. I've seen that pattern as well to not log an awful lot, but even after my little experience now dealing with kitsune problems, I'm all in favor for "more is better". We're nowhere near the tracing disaster that some of the holochain modules were.